website/data/plugins-manifest.json

```diff
@@ -214,6 +214,13 @@
     "repo": "hashicorp/packer-plugin-parallels",
     "version": "latest"
   },
+  {
+    "title": "Password",
+    "path": "password",
+    "repo": " alexp-computematrix/packer-plugin-password",
```

Comment:
Is there a reason why we have a blank in front of the repo name? `..." alexp...`

Reply:
Typo, addressed in latest commit.